Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -257,10 +257,16 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     effectSpecifiers: (some EffectSpecifiersSyntax)?,
     misplacedSpecifiers: UnexpectedNodesSyntax?
   ) {
+    var correctTokens = [effectSpecifiers?.asyncSpecifier, effectSpecifiers?.throwsClause?.throwsSpecifier]
+    if let throwingType = effectSpecifiers?.throwsClause?.type {
+      correctTokens.append(effectSpecifiers?.throwsClause?.leftParen)
+      correctTokens += Array(throwingType.tokens(viewMode: .all))
+      correctTokens.append(effectSpecifiers?.throwsClause?.rightParen)
+    }
```

Comment:
Couldn’t this be simplified to

```suggestion
    if let throwsClause = effectSpecifiers?.throwsClause {
      correctTokens += throwsClause.tokens(viewMode: .all)
    }
```